src/vivarium/framework/results/manager.py

```diff
@@ -301,7 +302,7 @@ def _bin_data(data: Union[pd.Series, pd.DataFrame]) -> pd.Series:
 
     def register_observation(
         self,
-        observation_type,
+        observation_type: Type[BaseObservation],
```

Comment:
This wasn't causing an error but it was just wrong. At this point we're sending in the class itself, NOT an instantiation of it (actually, it might be worth just saying `observation_type: Type[StratifiedObservation | UnstratifiedObservation | AddingObservation | ConcatenatingObservation]` if folks prefer b/c that's what it really is. But they all inherit from BaseObservation so this is less typing.

Reply:
Should `BaseObservation` be called `Observation`?

Reply:
I think we had it called that at once and changed it to this but now can't recall why. It does subclass from abc, though there are no abstract methods or anything on it. Probably doesn't matter?

Reply:
Type[C] is covaiant, so I think putting in the base observation makes sense. It looks like there is a builtin `type` you could use instead, though